### DISCUSSION TITLE
Create the same light setting as model-viewer in blender

### DISCUSSION THREAD

**Discussion by tian-yuyang** [google/model-viewer#4713] [Q&A] [answered]
Hi,
I'm facing a problem with the light location and the type of light I should use to render the same image as in blender as in model-viewer. Could you tell me more about the light setting and how to create the same image? 
Thanks a lot!

**Comment by elalish** [comment, marked as answer]
You can download our neutral lighting environment [here](https://github.com/google/model-viewer/blob/master/packages/shared-assets/environments/neutral.hdr). You'll also need the same tone mapping - I'd recommend tone-mapping="commerce" for MV (needs the name updated still) and these directions for Blender: https://modelviewer.dev/examples/tone-mapping#ocio

**Reply by jackblackCH** [reply]
What's the right way to set the same ecommerce tone mapping on the React Three Fiber Canvas?

**Reply by elalish** [reply]
Assuming R3F exposes the Three.js API, they now have `NeutralToneMapping`, which aligns with MV's "neutral" (new name for "commerce") and Blender 4.2's "Khronos PBR Neutral" view transform.

**Comment by tian-yuyang** [comment]
thanks for your help, I appreciate it.
